renderer/components/Settings/messages.js

```diff
@@ -63,4 +63,6 @@ export default defineMessages({
   payments_maxParts_label: 'Payment parts',
   payments_maxParts_description:
     'Maximum number of partial payments that may be used to send a payment.',
+  clockFormat24_label: 'Enable 24-hour clock',
```

Comment:
After adding new message strings, you should run `yarn extract-messages` in order to extract into the translation files.